packages/qwik-labs/src/qwik-types/public-api.ts

```diff
@@ -17,7 +17,15 @@ export const untypedAppUrl = function appUrl(
       path.splice(i, 1);
     }
   }
-  return path.join('/');
+  if (import.meta.env.BASE_URL) {
+    let baseURL = import.meta.env.BASE_URL;
```

Comment:
Actually, it's just a tiny bit shorter if you move this line to before the if block and test baseUrl

Reply:
fixed: https://github.com/BuilderIO/qwik/pull/5615/commits/78a50b5d81b4ee6249d905595c2eb1e356dffccc